build.gradle

```diff
@@ -15,15 +15,15 @@
  */
 
 plugins {
-    id 'com.gradle.build-scan' version '1.9' // declare before any other plugin
+    id 'com.gradle.build-scan' version '1.10.1' // declare before any other plugin
 
     id 'com.github.sherter.google-java-format' version '0.6'
     id 'com.github.johnrengelman.shadow' version '2.0.1' apply false
-    id 'me.champeau.gradle.jmh' version '0.4.4' apply false
-    id 'io.morethan.jmhreport' version '0.6.2.1' apply false
+    id 'me.champeau.gradle.jmh' version '0.4.5-beta-2' apply false
```

Comment:
I'd avoid betas.  Also can we buffer version bumps until there is something needed? There is some risk in moving ahead of transitive dependencies of reactor-core, so let's not always be on the bleeding edge.